packages/logrotate/build

```diff
@@ -21,9 +21,9 @@ nomail
 
 /var/lib/dcos/mesos/log/* {
     olddir /var/lib/dcos/mesos/log/archive
-    maxsize 2000k
-    daily
-    rotate 7
+    maxsize 256M
+    hourly
+    maxage 14
```

Comment:
`maxage` determines when files should be deleted.  `hourly` and `maxsize` (and `size`) determine when files should be rotated.